Generate PO file from Inno setup isl file.

mrpau-richard opened the pull request:
### Summary 
I created a Python script to create po file and extract translatable strings from Inno setup isl file.


### Issue addressed 
#29 


### TODO

- [x] Create to script to encode translated strings from po file to inno setup isl file.


/cc @radinamatic

mrpau-richard commented:
@radinamatic here's the [installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMZXJCMlkyUjBNTkk) to test the Spanish language translated from the crowdin.

I'm currently working on the documentation on how to extract and encode the strings from the Inno setup `isl` file.

radinamatic commented:
@mrpau-richard I'm going through the installer steps to see how can I improve the translation, but the main problem I see for now is codification: Spanish accented vowels do not display correctly:

![fullscreen_04_10_17_02_42](https://user-images.githubusercontent.com/1457929/31155523-38d39d10-a8af-11e7-838f-7ca67aa86c6d.png)

![fullscreen_04_10_17_02_45](https://user-images.githubusercontent.com/1457929/31155526-3ed0c120-a8af-11e7-8ac4-09b75356938e.png)

mrpau-richard commented:
@radinamatic I think that's an installer issue, let me fix that one.

mrpau-richard commented:
@radinamatic I've fixed the Spanish text that's not properly displaying. I also added some languages that's been translated in [Inno setup official translations](http://www.jrsoftware.org/files/istrans/).
Here's a sample screenshots:
![screen shot 2017-10-04 at 1 03 59 pm](https://user-images.githubusercontent.com/8663934/31167748-b50455ca-a925-11e7-9a07-12b7f1c3fdcd.png)
![screen shot 2017-10-04 at 1 03 39 pm](https://user-images.githubusercontent.com/8663934/31167755-bca3881e-a925-11e7-9dbc-9b2de817e9d0.png)

Here's the [installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMWi1GWThMeGc4eFU) for your testing.

radinamatic commented:
@mrpau-richard Ok, looking much better now, codification issues gone! :+1: 

On another hand, there are still a couple of problems:

1. Button strings in this window are not localized:

![fullscreen_04_10_17_12_13](https://user-images.githubusercontent.com/1457929/31172545-3fe66056-a904-11e7-81c3-31f570ba5fcd.png)

2. This installer is not merged with the latest enhancements, right? Since I can still see the steps that we made invisible to the user:

![fullscreen_04_10_17_12_17](https://user-images.githubusercontent.com/1457929/31172638-a4e8536a-a904-11e7-8b17-eabdb88b4474.png)

3. After the installation is done in Spanish, for example, the startup messages should be in that language  too. Right now they still appear in English:

![fullscreen_04_10_17_12_20](https://user-images.githubusercontent.com/1457929/31172807-5762cb74-a905-11e7-9118-04205d5c5ec9.png)

![fullscreen_04_10_17_12_21](https://user-images.githubusercontent.com/1457929/31172822-648b6fea-a905-11e7-9be5-a6f844176b43.png)

4. If the language used in the installation up to this point is present in Kolibri, the ideal would be to have it selected by default on this step too, although I'm not sure if you can pass that argument to the installer... :thinking: 

![fullscreen_04_10_17_12_34](https://user-images.githubusercontent.com/1457929/31172996-1618f8a4-a906-11e7-93f0-a7765cb465ec.png)

Maybe @indirectlylit and @rtibbles could chime on this too.

mrpau-richard commented:
@radinamatic the reason why those message box buttons is in english it's because your Windows language is in English. Here's the reference http://www.jrsoftware.org/iskb.php?translatebuttons

radinamatic commented:
Makes sense, @mrpau-richard!
One thing less to fix :+1:

mrpau-richard commented:
@radinamatic We can also use the selected language from the installer into GUI application and Kolibri itself. we just need to set the selected language into environment variable and we can use it in Kolibri by running this command `kolibri language setdefault lang` before starting the server.

indirectlylit commented on windows/installer-source/KolibriSetupScript.iss:
```diff
@@ -31,11 +31,19 @@ UsePreviousAppDir=yes
 [Languages]
-Name: "English"; MessagesFile: "compiler:default.isl"
+Name: "english"; MessagesFile: "compiler:Default.isl"
+Name: "spanish"; MessagesFile: "compiler:Languages\Spanish.isl"
```
curious why `english` and `spanish` are lowercase but the other languages are uppercase?

indirectlylit commented:
mind resolving merge conflicts and posting a new installer to test? thanks!

mrpau-richard commented:
@radinamatic & @indirectlylit here's the newly built [installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMU1Njd0VnME5tSUE) to test this PR with the update's from the develop branch.

indirectlylit commented:
looks like a (very minor) change to the readme got reverted in the merge:

![image](https://user-images.githubusercontent.com/2367265/31677087-91418d1c-b31e-11e7-80ae-0c4bf7fa4bf8.png)

no worries, I'll re-make the change
